patternfly/patternfly-elements
# Fix IE11 bugs in pfe-tabs

## Fix IE11 bugs in pfe-tabs

* Tabs have fallen upon hard times in IE11, this branch's only goal is to fix that so that they are more presentable again

<img width="860" alt="IE11-tab1" src="https://user-images.githubusercontent.com/1840295/67807168-a9936f80-fa6a-11e9-9400-12df0af8b037.png">

<img width="869" alt="IE11-tab2" src="https://user-images.githubusercontent.com/1840295/67807185-b0ba7d80-fa6a-11e9-8d59-9908de41cf63.png">

<img width="871" alt="IE11-tab3" src="https://user-images.githubusercontent.com/1840295/67807196-b6b05e80-fa6a-11e9-8056-c396b03f0b5f.png">

---

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* IE11 wasn't getting the variable updates so I added scoped IE11 queries with property style updates in parity with the variables being changed.

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. Pull down the branch and run `npm run demo`. 
2. Pull the tabs demo page up in IE11 to confirm fixes.
3. View demo page in other browsers to confirm no regressions.

#### Browser requirements
Your component should work in all of the following environments:


### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
